tests/integration/bundles/kfp_latest_edge.yaml.j2

```diff
@@ -1,13 +1,35 @@
 bundle: kubernetes
 name: kubeflow-pipelines
 applications:
-  argo-controller:         { charm: ch:argo-controller, channel: latest/edge, scale: 1 }
+  argo-controller:         { charm: ch:argo-controller, channel: latest/edge, scale: 1, trust: true }
   metacontroller-operator: { charm: ch:metacontroller-operator, channel: latest/edge, scale: 1, trust: true }
   minio:                   { charm: ch:minio, channel: latest/edge,       scale: 1 }
-  kfp-db:                  { charm: ch:mysql-k8s, channel: 8.0/stable, scale: 1, constraints: mem=2G}
+  kfp-db:                  { charm: ch:mysql-k8s, channel: 8.0/stable, scale: 1, constraints: mem=2G, trust: true }
+  mlmd:                    { charm: ch:mlmd, channel: latest/edge, scale: 1 }
+  envoy:                   { charm: ch:envoy, channel: latest/edge, scale: 1 }
+  kubeflow-profiles:       { charm: ch:kubeflow-profiles, channel: latest/edge, scale: 1, trust: true }
+  istio-ingressgateway:
+    charm: istio-gateway
+    channel: latest/edge
+    scale: 1
+    options:
+      kind: ingress
+    trust: true
+  istio-pilot:
+    charm: istio-pilot
+    channel: latest/edge
+    scale: 1
+    options:
+      default-gateway: kubeflow-gateway
+    trust: true
+  kubeflow-roles:
+    charm: kubeflow-roles
+    channel: latest/edge
+    scale: 1
+    trust: true
 {%- if local_build == false %}
   kfp-api:                 { charm: ch:kfp-api, channel: latest/edge, scale: 1, trust: true}
-  kfp-profile-controller:  { charm: ch:kfp-profile-controller, channel: latest/edge, scale: 1, trust: true }
+  kfp-metadata-writer:     { charm: ch:kfp-metadata-writer, channel: latest/edge, scale: 1, trust: true}
```

Comment:
I know we debated this a [a few months back](https://github.com/canonical/kfp-operators/pull/238#issuecomment-1624037301), but I'd still vote to keep `kfp-profile-controller` in here.  I don't think kfp functions completely without it, even if we don't have a test yet that spots the parts that are broken.  I prefer keeping our test env close to our prod env, even if we can't point to a specific reason for it.

If we do remove kfp-profile-controller from this bundle as proposed, then I think we should also remove `metacontroller-operator`.  `metacontroller-operator` is included only because it is a dependency of `kfp-profile-controller`

Reply:
I can add it back. We can increase our test coverage later if we want.

Reply:
I feel better with kfp-profile-controller in the test, but trust your judgement so whichever you choose is fine with me.  

But if you choose to remove kfp-profile-controller, then we should also remove metacontroller-operator